### PR TITLE
Bug 1881046: Add manifests and regenerate bindata

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: assets/credentials.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-openstack-cinder-csi-driver
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: openstack-cloud-credentials
+    namespace: openshift-openstack-cinder-csi-driver
```
Comment (bertinatto) [inline assets/credentials.yaml:9]: The NS should be `openshift-cluster-csi-drivers`

File: assets/credentials.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: OpenStackProviderSpec
```
Comment (bertinatto) [inline assets/credentials.yaml:12]: Recently we moved the CredentialsRequest creation to CVO. See https://github.com/openshift/cluster-storage-operator/blob/master/manifests/03_credentials_request_openstack.yaml

File: assets/namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-openstack-cinder-csi-driver
```
Comment (bertinatto) [inline assets/namespace.yaml:4]: The NS should be `openshift-cluster-csi-drivers` and it's created by CVO, so the CSI driver operator doesn't need to create it. You can add it to `manifests/` for future use, like the  AWS EBS CSI driver operator does: https://github.com/openshift/aws-ebs-csi-driver-operator/tree/master/manifests

File: assets/credentials.yaml (hunk 2 above)
Comment (Fedosin) [reply assets/credentials.yaml]: done

File: assets/namespace.yaml (hunk 3 above)
Comment (Fedosin) [reply assets/namespace.yaml]: done

File: pkg/generated/bindata.go [hunk 4]
```diff
@@ -2,9 +2,7 @@
 // sources:
 // assets/controller.yaml
 // assets/controller_sa.yaml
-// assets/credentials.yaml
 // assets/csidriver.yaml
-// assets/namespace.yaml
```
Comment (bertinatto) [inline pkg/generated/bindata.go:7]: Perhaps did you forget `make update`?

Comment (Fedosin) [reply pkg/generated/bindata.go:7]: no... why? we don't have these assets anymore, so these lines should be removed, right?
I tried to do `make update` again on this patch and nothing happened

**Comment by jsafrane** [comment]
looks good to me, @bertinatto?

File: pkg/generated/bindata.go (hunk 4 above)
Comment (bertinatto) [reply pkg/generated/bindata.go:7]: hm, I think _I_ forgot to `make update` in my first PR

**Review by bertinatto — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**